src/main/typescript/actor/wvActor.ts

```diff
@@ -476,58 +482,82 @@ export default class WvActor extends Actor {
     });
   }
 
+  async rollCreateCheckMessage(
```

Comment:
This is missing documentation.

Reply:
Also it could probably be renamed to be more clear. For example: "createCheckRollMessage"